lib/resolveExtension.js

```diff
@@ -53,6 +54,26 @@ exports.resolveExtension = async function ({ id, repository, location }, ms) {
   await exec(`git clone --filter=blob:none --recurse-submodules ${repository} ${repoPath}`, { quiet: true });
 
   const packagePath = [repoPath, location, 'package.json'].filter(p => !!p).join('/');
+
+  //#region Check if the extension has an OSI-approved open-source license
+  try {
```

Comment:
Should not we check a revision form which we build vsix file?

Reply:
Great catch, @akosyakov, I was thinking about doing that, settled on "licenses don't change from commit to commit, right?". In edge cases where a license might be changed, added, or removed, I'll integrate it into the `resolveVersion` function to avoid otherwise duplicating the code over and over.

https://github.com/open-vsx/publish-extensions/blob/4d657302ecdb5f219159486335ec883df1216bf9/lib/resolveExtension.js#L60-L71

Reply:
Just looked into this again, and it seems there is no way to fetch the license for a specific time in the repository, just the current state. I checked with both the REST and GraphQL APIs, the only parameters allowed are `name` and `owner`.